integration-templates/zoho-mail/nango.yaml

```diff
@@ -1,47 +1,124 @@
 integrations:
-  zoho-mail:
-    syncs:
-      zoho-mail-tasks:
-        runs: every 6 hours
-        description: |
-          Fetches a list of all your personal tasks in Zoho mail
-        output: ZohoMailTask
-        sync_type: full
-        endpoint: /zoho-mail/tasks
-        scopes:
-          - ZohoMail.tasks.READ
+    zoho-mail:
+        syncs:
+            zoho-mail-tasks:
+                runs: every 6 hours
+                description: |
+                    Fetches a list of all your personal tasks in Zoho mail
+                output: ZohoMailTask
+                sync_type: full
+                endpoint: /zoho-mail/tasks
+                scopes:
+                    - ZohoMail.tasks.READ
+            zoho-mail-emails:
+                runs: every 6 hours
+                description: |
+                    Fetches a list of all your account's emails in Zoho mail
+                metadata:
```

Comment:
This extra metadata object should be removed

Reply:
@khaliqgant , just updated my pull request. Quick one, when should we add the metadata object?

Reply:
You shouldn't really with the updated v2 version of the nango.yaml

Reply:
@khaliqgant thanks for the clarification 🙏.